### DISCUSSION TITLE
到底如何彻底弄掉ipv6啊

### DISCUSSION THREAD

**Discussion by GreatMichaelLee** [coolsnowwolf/lede#12239] [Bugs / 问题反馈]
我编译lede固件，能关掉的相关组件make menuconfig里都去掉了，系统里相关配置也全部都按网上教程关掉了，光猫也去使能了ipv6,可是为啥每次这些wan/lan接口还是有外部ipv6地址获得啊？见了个鬼啊，谁能告诉我怎么搞？

![image](https://github.com/coolsnowwolf/lede/assets/20067401/29bcb67c-4e58-4e6a-a4ee-b993f666250d)

**Comment by xiaouex** [comment]
extra package里关掉 ipv6 helper和base system里去掉ipv6 dhcp支持

**Reply by GreatMichaelLee** [reply]
> extra package里关掉 ipv6 helper和base system里去掉ipv6 dhcp支持

说了都做了，不过我搞定了，通过这个方式

echo 'net.ipv6.conf.all.disable_ipv6 = 1' >> /etc/sysctl.conf
sysctl -p /etc/sysctl.conf

**Reply by xiaouex** [reply]
> > extra package里关掉 ipv6 helper和base system里去掉ipv6 dhcp支持
> 
> 说了都做了，不过我搞定了，通过这个方式
> 
> echo 'net.ipv6.conf.all.disable_ipv6 = 1' >> /etc/sysctl.conf sysctl -p /etc/sysctl.conf

我的情况和你正好相反，我是不加这两个，死活拿不到ipv6地址
